airflow/serialization/serialized_baseoperator.py

```diff
@@ -89,7 +96,10 @@ def deserialize_operator(cls, encoded_op: dict) -> BaseOperator:
                         operator.__module__ == encoded_op["_task_module"]:
                     op_extra_links_from_plugin.update({ope.name: ope})
 
-        setattr(op, "operator_extra_links", list(op_extra_links_from_plugin.values()))
+        # If OperatorLinks is defined in Plugins but not in the Operator
+        # set the Operator links attribute
+        if op_extra_links_from_plugin and "_operator_links_sources" not in encoded_op:
+            setattr(op, "operator_extra_links", list(op_extra_links_from_plugin.values()))
```

Comment:
What if we define Links in a plugin _and_ in the operator?

I'm finding this code (including the existing code) a bit hard to follow. Hmmmm.

Reply:
If we define in a plugin and in operator, then the following code get's executed:
https://github.com/apache/airflow/pull/6715/files#diff-f58748b66b2d3d00c8132103faea223fR114-R121

I can add more detailed comments if it is not clear.